### ISSUE TITLE
DNSimple Plugin not removing dns challange

### ISSUE THREAD

**Issue by jamessiuda1** [rmbolger/Posh-ACME#548]
posh acme is working great for us but it doesn't remove the txt record after the certificate is issued. Looking in debug mode it's stating the record does not exist, but it's still left after posh-acme is done running. Is this a bug with the plugin?

```powershell
VERBOSE: Unpublishing challenge for Domain testing.example.com with Token TkpiNJXUMbszHlI9vdjRJ1N4ROJJYkG277q8EwpBg5s using Plugin DNSimple and DnsAlias ''.
DEBUG: Loading PAAccount list from disk
DEBUG: Calling DNSimple plugin to remove _acme-challenge.testing.example.com TXT with value Wgya1aoWyRSBcpztFdj1NEWzM0gUvOLEMzqk1VsIDPA
VERBOSE: Requested HTTP/1.1 GET with 0-byte payload
VERBOSE: Received HTTP/1.1 response of content type application/json of unknown size
VERBOSE: Content encoding: utf-8
DEBUG: Found account 87989
DEBUG: Found zone example.com
VERBOSE: Requested HTTP/1.1 GET with 0-byte payload
VERBOSE: Received HTTP/1.1 response of content type application/json of unknown size
VERBOSE: Content encoding: utf-8
DEBUG: Record _acme-challenge.testing.example.com with value Wgya1aoWyRSBcpztFdj1NEWzM0gUvOLEMzqk1VsIDPA doesn't exist. Nothing to do.


```

**Comment by rmbolger** [comment]
Hey @jamessiuda1, thanks for reaching out. It may very well be a bug. I'll have to brush off my account and do some testing.

**Comment by rmbolger** [comment]
Bug confirmed. Not sure if the API changed or it has always been this way. But the code that was looking for the existing record needed to look for the TXT value with quotes around it and it hadn't been. I should have a basic fix committed to the main branch shortly.

There are a few other improvements I may try and make to the plugin as well. So it may be another couple days before I push an updated release.

**Comment by rmbolger** [comment]
Fix is now live in 4.23.0
